lib/matplotlib/table.py

```diff
@@ -243,6 +245,9 @@ def get_path(self):
             readonly=True
             )
 
+@cbook.deprecated('3.1', 'CustomCell functionality merged into Cell')
```

Comment:
```suggestion
@cbook.deprecated('3.2', 'CustomCell functionality merged into Cell')
```
3.1 is already out. We can only start deprecating for the next minor release.